back/config/initializers/que.rb

```
@@ -7,3 +7,14 @@
   config.retry_interval      = proc { |count| (count**6) + 15 + (rand(30) * (count + 1)) }
   config.maximum_retry_count = 9
 end
+
+# Latest version of Que didn't add this method yet.
+Rails.application.config.to_prepare do
+  if defined?(ActiveJob::QueueAdapters::QueAdapter)
+    ActiveJob::QueueAdapters::QueAdapter.class_eval do
+      def enqueue_after_transaction_commit?
+        false
+      end
+    end
+  end
+end
```

I think I know what this is about, but I think it deserves a clearer explanation for our future selves. Also, ideally, it should fail once Que adds the missing method.